llvm/lib/Target/PowerPC/PPCPreEmitPeephole.cpp

```diff
@@ -109,6 +109,93 @@ static bool hasPCRelativeForm(MachineInstr &Use) {
           MachineFunctionProperties::Property::NoVRegs);
     }
 
+    // The funtion will simply the zeroing accumulator and spilling instrcutions
+    // into simple xxlxor and spilling instrcuctions.
+    // From:
+    // setaccz acci
+    // xxmfacc acci
+    // stxv vsr(i*4+0), D(1)
```

Comment:
What does `D(1)` mean? 1 is for R1? could we use another name like `Base`? And we don't check the offsets in the implementation, so maybe the D/D-16/D-32/D-48 is also not necessary?